httpcore5/src/test/java/org/apache/hc/core5/net/TestURIBuilder.java

```diff
@@ -171,7 +171,7 @@ void testFormatQuery() {
     @Test
     void testHierarchicalUri() throws Exception {
         final URI uri = new URI("http", "stuff", "localhost", 80, "/some stuff", "param=stuff", "fragment");
-        final URIBuilder uribuilder = new URIBuilder(uri);
+        final URIBuilder uribuilder = new URIBuilder(uri).setEncodingPolicy(URIBuilder.EncodingPolicy.ALL_RESERVED);
```

Comment:
@arturobernalg I think this should no longer be necessary. `ALL_RESERVED` is default now.

Reply:
done